test/relations.test.js

```diff
@@ -4987,7 +4987,7 @@ describe('relations', function() {
       Category.findById(category.id, function(err, cat) {
         if (err) return done(err);
         var link = cat.items.at(0);
-        link.updateAttributes({notes: 'Updated notes...'}, function(err, link) {
```

Comment:
A slight change, now it is `SQL.prototype.updateAttribute()` in pr https://github.com/strongloop/loopback-connector/pull/100 instead of `SQL.prototype.update()` in pr https://github.com/strongloop/loopback-connector/pull/97. That does not change much, but just for the record. 

There is the `Category` that embedsMany `items`, so there should be 2 queries, but  It seems like only 3 connectors `mssql`, `oracle`, `postgresql` are not recognizing the 2nd query. They only execute the one on `items` and obviously they do not see it in the database, so there are 2 options:
1) skip the test for the three connectors
2) this solution implemented in this pr, which is to use `updateById` which is one of the methods of `embedsMany` to fix the situation. https://loopback.io/doc/en/lb2/Embedded-models-and-relations.html#embedsmany